app/src/components/custom/modals/DeletePrompt.tsx

```diff
@@ -0,0 +1,76 @@
+import { bcgov } from '../../../constants/colours';
+import ActionButton from '../../bcgov/ActionButton';
+import { buttonStyles } from '../../bcgov/ButtonStyles';
+
+/**
+ * @interface
+ * @description Properties passed to the DeletePrompt component.
+ * @property {() => void} deleteHander - The action taken when delete is confirmed.
+ * @property {string} title - The title of the modal popup.
+ * @property {string} blurb - The text in the body of the modal. If multiple paragraphs are required, separate the paragraphs with ";;".
+ * @property {string} id - The id assigned to this modal.
+ */
+interface DeletePromptProps {
+  deleteHandler: () => void;
+  title: string;
+  blurb: string;
+  id: string;
+}
+
+/**
+ * @description A modal element that can be used to confirm deletions.
+ * @param {DeletePromptProps} props The properties passed to the component.
+ * @returns A React component.
+ */
+const DeletePrompt = (props: DeletePromptProps) => {
+  const { deleteHandler, title, blurb, id } = props;
+  return (
+    <dialog
```

Comment:
I like the use of the dialog element as it's had some big improvements recently.